actionview/test/template/tag_helper_test.rb

```diff
@@ -426,6 +421,16 @@ def test_content_tag_with_unescaped_conditional_hash_classes
     assert_equal "<p class=\"song play>\">limelight</p>", str
   end
 
+  def test_content_tag_with_invalid_html_tag
+    invalid_tags = ["12p", "", "image file", "div/", "my>element", "_header"]
+    test_invalid_html_tags(invalid_tags) { |tag| content_tag(tag) }
```

Comment:
I don't think the `test_invalid_html_tags` helper method is necessary or makes this test more readable. I'd prefer to see something like:

```suggestion
    invalid_tags = ["12p", "", "image file", "div/", "my>element", "_header"]
    invalid_tags.each do |tag_name|
      assert_raise(ArgumentError, "expected #{tag_name.inspect} to be invalid") do
        content_tag(tag_name)
      end
    end
```